production/docker/config/loki.yaml

```diff
@@ -71,19 +70,13 @@ ruler:
 schema_config:
   configs:
   - from: 2020-08-01
-    store: boltdb-shipper
```

Comment:
I don't think we can change this in this way; folks may already be using this compose setup, and then their indexes won't work. We'll need to add a new period config from a new date.

Reply:
I can add as a new config, but honestly nothing we provide is going to be guaranteed to work out of the box. If I provide a date the in the future, it's going to require the stack to be running when the changeover happens. If I provided a date in the past, then chunks/indexes already written past that time will no longer work.

Maybe the safest thing to do is add with the approximate date of this PR being merged, but it's still likely to break anyone using this who tries to use our config out of the box.

Reply:
> but it's still likely to break anyone using this who tries to use our config out of the box

I don't understand why that would be the case?

Reply:
Re-reading your comment, I think you're right actually @trevorwhitney.
I still think we should add another period config, and then folks will just be able to change the date to get their storage config working again; replacing the store config will make that harder.

Reply:
sounds good, created a new schema starting on `2023-07-11`.